numba/tests/test_auto_constants.py

```diff
@@ -27,6 +29,12 @@ def pyfunc():
 
         self.assertEqual(pyfunc(), cfunc())
 
+    def test_module_string_constant(self):
+        @njit
+        def f():
+            return uc._GLOBAL_STR
+        self.assertEquals(f(), "abc")
```

Comment:
```suggestion
        self.assertEquals(f(), f.py_func())
```
saves keeping in sync